### ISSUE TITLE
Add name element to search inputs

### ISSUE THREAD

**Issue by mjoyce91** [18F/State-TalentMAP#322]
Pa11y and WAVE did not detect this, but HTML_CodeSniffer (which pa11y uses) detected that search inputs did not have a `name` element. This appears to be a false positive for the primary search field on the home route, but not for the new search input that was included in the header. A `name` element should be added to the text input in the header.

**Comment by mjoyce91** [comment]
This appears to be a false positive for the header as well.
